package.json

```diff
@@ -23,7 +23,11 @@
     "docs:build": "npm run docs:prepare && gitbook build -g redux-saga/redux-saga",
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && cp CNAME _book && touch _book/.nojekyll && gh-pages -d _book -r git@github.com:redux-saga/redux-saga.git -m 'update book' -t",
-    "bundlesize": "lerna run --parallel bundlesize"
+    "bundlesize": "lerna run --parallel bundlesize",
+    "changeset": "changeset",
+    "version-packages": "changeset version",
```

Comment:
with a changeset bot is there any need for this script? versioning is done automatically by a bot, right?

Reply:
With the bot, there's no need for it.